### DISCUSSION TITLE
Задание 3

### DISCUSSION THREAD

**Discussion by ArtemDzyachenka** [ArtemDzyachenka/Amasty_ArtemModuleex1#15] [General]
Задание 3, разбор ошибок

**Comment by AmastyVadim** [comment]
**Оценка**: 6

Замечания оставил в пул реквестах.
___
Старайтесь загружать задание одним пул реквестом. Ваше третье задание разбито на 7 реквестов. Это очень сложно читается и существенно усложняет проверку вашего задания. Т.к. приходится бегать по всем этим реквестам и проверять изменения в каждом из них, прежде чем что-то написать вам в дискашене.
